cookbooks/cdo-users/templates/default/bashrc.erb

```diff
@@ -57,9 +57,9 @@ if [ -n "$force_color_prompt" ]; then
 fi
 
 if [ "$color_prompt" = yes ]; then
-    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@<%= node.name %>\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
 else
-    PS1='${debian_chroot:+($debian_chroot)}\u@<%= node.name %>:\w\$ '
+PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
```

Comment:
Random low-risk cleanup here- since our Chef nodes are identical to the hostname, I thought we should use the hostname directly here rather than `node.name` through the Chef template.